buildSrc/build.gradle.kts

```diff
@@ -20,9 +20,9 @@ plugins {
 
 repositories {
   google()
-  jcenter()
   gradlePluginPortal()
   maven("https://www.jitpack.io")
+  mavenCentral()
```

Comment:
The order of the declarations is the order where Gradle will try to search for the artifacts.
`mavenCentral` will replace `jcenter`, and so represents the main repository (even above `google` I guess).
Could you put it at the top of the declaration please ?

For more info: https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:declaring_multiple_repositories